kernel/power/gemm_common.c

```diff
@@ -28,7 +28,12 @@
 #define USE_VECTOR_PAIRS
 #endif
 
+#ifdef _AIX
```

Comment:
I think this should be `#if defined(_AIX) && defined(__clang__)` since you are saying it only breaks on OpenXL